veeqtor/API-Trisixty
# #164350465 Endpoint for vendor owners to create and edit products

#### What does this PR do?
- Implement the view for creating a vendor.

#### Description of Task to be completed?
As a verified, logged in business account owner, I should be able to create a product for sell under my vendor name by making a `POST` request to this route `/product`.

#### How should this be manually tested?
- Fetch the branch with `git fetch origin ft-create-products-164350465`
- Checkout to the branch with `git checkout ft-create-products-164350465`
- Activate the virtual environment and start the app with `pipenv shell && python manage.py runserver`
- Make a post request to the route `/product` with the body as follows:

`POST /product`

```
{
    "title": "<product_title>",
    "price": <product_price>,
    "images": [<image_urls>, ... ],
    "description": "<product description>"
    "vendor_id": "<vendor_id>"
}
```

RESPONSE
`201 CREATED`

Response data should have the following structure
```
{
    "message" : "Product successfully created",
    "status" : "success"
}
```

**NOTE** 
You must be an authenticated and verified business account owner to perform the actions.

#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?
- [#164350465](https://www.pivotaltracker.com/story/show/164350465)

#### Screenshots (if appropriate)
- N/A

#### Questions:
 - N/A
